aws/eks/eks.tf

```diff
@@ -49,13 +49,14 @@ module "vpc" {
 
 module "eks" {
   source  = "terraform-aws-modules/eks/aws"
-  version = "18.2.0"
+  version = "20.4.0"
 
   cluster_name    = "${var.app}-${var.environment}"
   cluster_version = var.cluster_version
 
-  vpc_id     = module.vpc.vpc_id
-  subnet_ids = concat(module.vpc.private_subnets, module.vpc.public_subnets)
+  vpc_id                                       = module.vpc.vpc_id
+  subnet_ids                                   = concat(module.vpc.private_subnets, module.vpc.public_subnets)
+  node_security_group_enable_recommended_rules = false
```

Comment:
> The new variable node_security_group_enable_recommended_rules is set to true by default and may conflict with any custom ingress/egress rules. Please ensure that any duplicates from the node_security_group_additional_rules are removed before upgrading, or set node_security_group_enable_recommended_rules to false. [Reference](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/UPGRADE-19.0.md#added)

> Support for setting default node security group rules for common access patterns required:
> - Egress all for 0.0.0.0/0/::/0
> - Ingress from cluster security group for 8443/TCP and 9443/TCP for common applications such as ALB Ingress Controller, Karpenter, OPA Gatekeeper, etc. These are commonly used as webhook ports for validating and mutating webhooks

I disabled it because I'm not sure we need those default rules and we specify a custom egress rule.